schemas/vstats.json

```diff
@@ -289,7 +289,6 @@
                 "title": "Add extra disk",
                 "default": false,
                 "description": "Add a second disk to separate Elasticsearch data from primary image. Disk is added to path /var/lib/elasticsearch on the VSTAT. Supported on KVM only.",
-                "target_server_type": "kvm",
```

Comment:
Since Schema files have changed, the examples need to be regenerated. I will push the change to regenerate the example files